libraries/cms/html/contentlanguage.php

```diff
@@ -44,7 +70,8 @@ public static function existing($all = false, $translate = false)
 			$query = $db->getQuery(true);
 
 			// Build the query.
-			$query->select('a.lang_code AS value, a.title AS text, a.title_native')
+			$query->select('a.lang_code AS value, a.title AS text, a.title_native, a.lang_id AS lang_id')
```

Comment:
why use the table alias "a." if you only use one table?
Suggestion: all field should have a ```$db->quoteName('xxxxx')```